job.go

```diff
@@ -24,6 +24,7 @@ type Job struct {
 	Args  []interface{} `json:"args"`
 
 	// optional
+	Priority   uint8                  `json:"priority"`
```

Comment:
Should this be omitempty also, to minimize JSON size on disk/network, since we have a default?

Reply:
Perhaps all access to the job's priority should be via an accessor, Priority(), so we can always return 5 as default and minimize JSON size?